### ISSUE TITLE
🐛 CSS `contrast-color` reported as unknown function

### ISSUE THREAD

**Issue by tim-we** [biomejs/biome#10536]
### Environment information

<details>

```bash

CLI:
  Version:                      2.4.16
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           windows

Environment:
  BIOME_LOG_PATH:                    unset
  BIOME_LOG_PREFIX_NAME:             unset
  BIOME_LOG_LEVEL:                   unset
  BIOME_LOG_KIND:                    unset
  BIOME_CONFIG_PATH:                 unset
  BIOME_THREADS:                     unset
  BIOME_WATCHER_KIND:                unset
  BIOME_WATCHER_POLLING_INTERVAL:    unset
  NO_COLOR:                     unset
  TERM:                         unset
  JS_RUNTIME_VERSION:           v26.2.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         npm/11.13.0

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.jsonc
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  false
  HTML full support enabled:    unset

Workspace:
  Open Documents:               0

```

</details>

### What happened?

Using the CSS function `contrast-color` triggers the rule `correctness/noUnknownFunction`:

https://biomejs.dev/playground/?indentStyle=space&attributePosition=multiline&bracketSameLine=true&indentScriptAndStyle=true&lintRules=noUnknownFunction&tab=formatter&pane=Diagnostics&code=LgBmAG8AbwAgAHsACgAgACAAYwBvAGwAbwByADoAIABjAG8AbgB0AHIAYQBzAHQALQBjAG8AbABvAHIAKAByAGUAZAApADsACgB9AAoA&language=css

### Expected result

`contrast-color` is available in all major browsers (Baseline 2026) and should therefore not be an unknown function.

https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Values/color_value/contrast-color

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by tim-we** [comment]
@siketyan I see you added `S-God first issue` and `S-Help-wanted` labels. I am happy to give this a go. Can you add me as an assignee?

**Comment by siketyan** [comment]
@tim-we All yours!
